### PR TITLE
new services tool

### PR DESCRIPTION
**Pull request by jhsimpson**
and update restart-services to work with Archivmatica 1.6.0

This commit has been tested with Ubuntu 14.04 (upstart).
TODO: support systemd for 16.04 and RedHat, will be added lin another pull request later.

### CODE REVIEW

**Comment by sevein** [comment]
You may want to look at implementing `tool/services` in Python - perhaps overkilling but let me try to explain. You could try to borrow the [service core module](https://github.com/ansible/ansible-modules-core/blob/devel/system/service.py) in Ansible - it has a really good abstraction of service management and it works with multiple init systems and platforms (implemented as subclasses). That’s the code used when you write a task that makes use of the service module, e.g. `service: state=reloaded`, etc...

**Comment by jhsimpson** [comment]
I think that's a good idea, I added a comment on issue #36 , with a milestone of 0.3.0, so that can be worked on after the Archivematica 1.6.0 release.  I am hoping this PR is sufficient for the 1.6.0 release.

**Comment by sevein** [comment]
I agree! 👍

File: tools/restart-services [hunk 1]
```diff
@@ -17,27 +17,52 @@
...
+if [ $VERSION == "1.5" ]; then
+    echo "If you are trying to use this command with Archivmeatica 1.6.0 or later,"
+    echo "there is a new command 'am services', try 'am services help' for more info."
+    echo " "
+    restart_services
+else
+    {
+        echo "try using the newer 'am services' command, see 'am services help' for more info."
+        echo "trying with $2"
+        [ -z "$2" ] && am services $2
```
Comment (jrwdunham) [inline tools/restart-services]: I don't think this line makes sense, but maybe I'm misunderstanding the intent. It seems like the intent is to allow a user to call `am restart-services 1.6 stop` and have that trigger the calling of `am services stop`. But that's not what happens: `am services restart` gets called every time.

Comment (jhsimpson) [inline tools/restart-services]: That wasn't the original intention.  restart-services should probably not support stop or start or status, it is confusing given the name 'restart-services'.

I added a commit to simplify this, restart-services will only try to restart, if you want to stop or start or status, use the new am services tool.

**Comment by jrwdunham** [comment]
👍

**Comment by sevein** [comment]
I think this was a good candidate to squash before merging, maybe next time?

**Comment by jhsimpson** [comment]
yes, probably could have been squashed.  Next time for sure.


Justin Simpson
Director of Archivematica Technical Services
www.artefactual.com
604-527-2056

**Comment by Hwesta** [comment]
This should update the README with info on the new tool and changes to the existing tool.

**Comment by jhsimpson** [comment]
yes, good point.  I will create a new PR for that?

**Comment by jhsimpson** [comment]
I just looked and the original tool (restart-services) was never added to the README, so I will add a description for both that and the new services tool.

**Comment by Hwesta** [comment]
Great, and a new PR would be best. Thanks!

**Pull request merged**